crates/eframe/Cargo.toml

```diff
@@ -183,7 +183,10 @@ objc2-app-kit = { version = "0.2.0", features = [
 
 # windows:
 [target.'cfg(any(target_os = "windows"))'.dependencies]
-winapi = { version = "0.3.9", features = ["winuser"] }
+windows-sys = { version = "0.59", features = [
```

Comment:
winit is still on 0.52. Might be better to sync with them.

Reply:
I see `cargo-deny` also complains about that. Version 0.52 has a less ergonomic API. I added a separate commit to make the difference visible. I will look into what it takes to update `winit` to version 0.59.



```
cargo deny --all-features --log-level error --target i686-pc-windows-gnu check 
error[duplicate]: found 2 duplicate entries for crate 'windows-sys'
    ┌─ C:\Users\caspe\Documents\GitHub\egui/Cargo.lock:275:1
    │
275 │ ╭ windows-sys 0.52.0 registry+https://github.com/rust-lang/crates.io-index
276 │ │ windows-sys 0.59.0 registry+https://github.com/rust-lang/crates.io-index
    │ ╰────────────────────────────────────────────────────────────────────────
```